src/pseudopeople/entity_types.py

```diff
@@ -76,6 +77,10 @@ def __call__(
             column, noise_level, randomness_stream, f"{self.name}_{additional_key}"
         )
         if to_noise_idx.empty:
+            logger.info(
+                f"No cells chosen to noise for noise function {self.name} on column {column.name}."
+                "This is likely due to a combination of the configuration noise levels and the input data."
```

Comment:
Hmm, this seems too nebulous. Maybe "This is likely due to a very low noise probability and/or a small number of input data that has potential to be noised".

Reply:
Let's let the RT decide what sort of language they'd like. This is fine for now.